templates/index.phtml

```diff
@@ -271,6 +271,12 @@
 		}
 
 		function clearFiles() {
+			<?php if(Webgrind_Config::$promptConfirmationBeforeDeleteFiles)?>
+			{
+			if (!window.confirm('Are you sure you want to clear all files?')) {
+				return;
+			}
+			<?php } ?>
```

Comment:
This is causing me a syntax error: `Parse error: syntax error, unexpected '}' in [...]`

Reply:
it should be 
``` php
			<?php if(Webgrind_Config::$promptConfirmationBeforeDeleteFiles): ?>
			if (!window.confirm('Are you sure you want to clear all files?')) {
				return;
			<?php endif; ?>
```